Source: https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/testing/element-desktop/tasje-one-hak.patch
directories in .hak/hakModules are already symlinked inside node_modules,
and as such are already being copied by default. this makes tasje fail with:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value:
FileAlreadyWritten("/node_modules/keytar/package.json")', src/main.rs:200:18
```

--- a/element-desktop/electron-builder.ts.orig
+++ b/element-desktop/electron-builder.ts
@@ -74,10 +74,6 @@
     },
     files: [
         "package.json",
-        {
-            from: ".hak/hakModules",
-            to: "node_modules",
-        },
         "lib/**",
     ],
     extraResources: [
